src/net.rs

```diff
@@ -7,24 +7,52 @@ use awc::ws::Codec;
 use awc::BoxedSocket;
 use std::ops::Not;
 
+pub const NET_URL_ENV_VAR: &str = "YAGNA_NET_URL";
+
 pub type Result<T> = std::result::Result<T, Error>;
 
 /// Bindings for Requestor part of the Net API.
 #[derive(Clone)]
-pub struct NetRequestorApi {
+pub struct NetApi {
+    client: WebClient,
+}
+
+impl WebInterface for NetApi {
+    const API_URL_ENV_VAR: &'static str = "YAGNA_NET_URL";
+    const API_SUFFIX: &'static str = ya_client_model::net::NET_API_V2_NET_PATH;
+
+    fn from_client(client: WebClient) -> Self {
+        NetApi { client }
+    }
+}
+
+impl NetApi {
+    /// Retrieves connection info.
+    pub async fn get_info(&self) -> Result<Info> {
+        self.client.get("info").send().json().await
+    }
+}
```

Comment:
`info` is very general name. I would consider `status`.
Check result of command `yagna net status`. I think we could return the same information.
(You can look at `StatusResponse` in `yagna/core/model/src/net`)

Reply:
Agree. Wrote just before in comment https://github.com/golemfactory/ya-client/pull/121#issuecomment-1168533694

Reply:
I'm against skipping it